mira/dkg/wsgi.py

```diff
@@ -16,14 +20,15 @@
 from mira.dkg.utils import PREFIXES, MiraState
 from mira.metamodel import RefinementClosure
 
-logger = logging.getLogger(__name__)
-
-
 __all__ = [
     "flask_app",
     "app",
 ]
 
+logger = logging.getLogger(__name__)
+
+HERE = Path(__file__).parent.resolve()
+EMBEDDINGS_PATH = HERE.joinpath("embeddings.tsv.gz")
```

Comment:
@kkaris how is the best way to pass through the path to this file from the docker configuration?

Reply:
This is resolved as of https://github.com/gyorilab/mira/pull/261/commits/1ea0b0dde8de9149628de964983882ebd39dcf2b